### ISSUE TITLE
Ran this with the new-vpc template from scratch, but can't ssh into bastion host

### ISSUE THREAD

**Issue by aaronvg** [aws-ia/cfn-ps-clickhouse-cluster#20]
I can't even ping my bastion host using its public IP. It seems perhaps the security group is wrong?

**Comment by qiaow02** [comment]
Yes, you need to update the inbound rules of the security group to allow your client IP or 0.0.0.0 to access bastion host as following:
<img width="478" alt="image" src="https://github.com/aws-ia/cfn-ps-clickhouse-cluster/assets/79438022/4f7e5b68-32ac-4b5a-b419-c27d09da7883">

<img width="1293" alt="image" src="https://github.com/aws-ia/cfn-ps-clickhouse-cluster/assets/79438022/794af836-ef9d-4542-a1ad-b0212b2535a1">
